content/en/docs/instrumentation/net/getting-started.md

```diff
@@ -88,187 +134,148 @@ following:
 <summary>View example output</summary>
 
 ```text
-Activity.Id:          00-d72f7e51dd06b57211f415489df89b1c-c8a394817946316d-01
-Activity.ParentId:    00-d72f7e51dd06b57211f415489df89b1c-e1c9fde6c8f415ad-01
-Activity.ActivitySourceName: MyCompany.MyProduct.MyServiceActivity.DisplayName: SayHello
-Activity.Kind:        Internal
-Activity.StartTime:   2021-12-21T01:15:27.5712866Z
-Activity.Duration:    00:00:00.0000487
-Activity.TagObjects:
+Activity.TraceId:            47d25efc8b5e9184ce57e692f5f65465
+Activity.SpanId:             bb864adcf4592f54
+Activity.TraceFlags:         Recorded
+Activity.ParentSpanId:       acbff23f5ad721ff
+Activity.ActivitySourceName: MyService
+Activity.DisplayName:        SayHello
+Activity.Kind:               Internal
+Activity.StartTime:          2023-02-21T12:27:41.9596458Z
+Activity.Duration:           00:00:00.0005683
+Activity.Tags:
     foo: 1
     bar: Hello, World!
-    baz: [1, 2, 3]
-Resource associated with Activity:
-    service.name: MyCompany.MyProduct.MyService
-    service.version: 1.0.0
-    service.instance.id: 45aacfb0-e117-40cb-9d4d-9bcca661f6dd
-
-Activity.Id:          00-d72f7e51dd06b57211f415489df89b1c-e1c9fde6c8f415ad-01
-Activity.ActivitySourceName: OpenTelemetry.Instrumentation.AspNetCore
-Activity.DisplayName: /hello
-Activity.Kind:        Server
-Activity.StartTime:   2021-12-21T01:15:27.5384997Z
-Activity.Duration:    00:00:00.0429197
-Activity.TagObjects:
-    http.host: localhost:7207
-    http.method: GET
-    http.target: /hello
-    http.url: https://localhost:7207/hello
-    http.user_agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.110 Safari/537.36
-    http.status_code: 200
-    otel.status_code: UNSET
+    baz: [1,2,3]
 Resource associated with Activity:
-    service.name: MyCompany.MyProduct.MyService
-    service.version: 1.0.0
-    service.instance.id: 45aacfb0-e117-40cb-9d4d-9bcca661f6dd
+    service.name: MyService
+    service.instance.id: 2b07a9ca-29c4-4e01-b0ed-929184b32192
 ```
 
 </details>
 
-This output has both the manually created span to track work in the route, and a
-span created by the `OpenTelemetry.Instrumentation.AspNetCore` instrumentation
-library that tracks the inbound ASP.NET Core request.
+You'll notice the `Activity` objects from ASP.NET Core alongside the `Activity`
+we created manually in our controller action.
 
-### Metrics
+### Automatic Metrics
```

Comment:
nit: use consistent headings. Automatic Metrics, Automatic Traces etc. or Metrics using Instrumentation Library, Traces using Instrumentation Library.

Reply:
Sure, the headings were a bit slapdash to be honest.